src/entities/AccessToken.ts

```diff
@@ -0,0 +1,33 @@
+import moment from "moment";
```

Comment:
If we are going to be using moment in the "shippable" code - we'll need to add it to regular dependencies `yarn add moment` not just the dev dependencies

Reply:
@nbragdon Fixed before your comment, but hadn't pushed up yet. Also added axios in the same way.